docs/core/diagnostics/dotnet-counters.md

```diff
@@ -417,7 +417,8 @@ dotnet-counters monitor [-h|--help] [-p|--process-id] [-n|--name] [--diagnostic-
 
 ## dotnet-counters ps
 
-Display a list of dotnet processes that can be monitored.
+ Lists the dotnet processes that cam be monitored.
+ dotnet-counters version 7 and later versions, also display the command-line arguments that each process was started with, if available.
```

Comment:
```suggestion
 dotnet-counters version 6.0.320703 and later, also display the command-line arguments that each process was started with, if available.
```

As far as I am aware there is no dotnet-counters version 7? Whatever we do for this one should also be repeated wherever else this pattern shows up for any of the tools.